app/gateway/traffic-control/proxy.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Proxying with {{site.base_gateway}}
+
+description: "Proxying is when {{site.base_gateway}} matches an HTTP request with a [registered route](/gateway/entities/route/) and forwards the request."
+
+content_type: reference
+layout: reference
+
+products:
+  - gateway
+
+related_resources:
+  - text: Route entity
+    url: /gateway/entities/route/
+  - text: Router Expressions language
+    url: /gateway/routing/expressions/
+  - text: Expressions repository
+    url: https://github.com/Kong/atc-router
+
+breadcrumbs:
+  - /gateway/
+---
+
+{{ page.description }} This page details information about how {{site.base_gateway}} handles proxying. The following diagram shows how proxying is handled by {{site.base_gateway}}:
+
+{% mermaid %}
+sequenceDiagram
+    actor Client
+    participant Gateway as Kong Gateway
+    participant Router
+    participant Plugins as Plugins
+    participant LoadBalancer as Load balancer
+    participant UpstreamService as Upstream Service
+
+    Client->>Gateway: Sends HTTP request or L4 connection
+    Gateway->>Router: Evaluates incoming request against Routes
+    Router->>Router: Orders Routes by priority
+    Router->>Gateway: Returns highest priority matching Route
+    Gateway->>Plugins: Executes plugins the `access` phase
+    Gateway->>LoadBalancer: Implements load balancing capabilities
+    LoadBalancer->>LoadBalancer: Distributes request across upstream service instances
+    LoadBalancer->>UpstreamService: Forwards request to selected instance
+    UpstreamService->>Gateway: Sends response
+    Gateway->>Plugins: Executes plugins in the `header_filter` phase
+    Gateway->>Client: Streams response back to client
+{% endmermaid %}
+
+{{site.base_gateway}} handles proxying in the following order:
+
+1. {{site.base_gateway}} listens for HTTP traffic on its configured
+proxy port(s) (`8000` and `8443` by default) and L4 traffic on explicitly configured
+`stream_listen` ports.
+1. {{site.base_gateway}} will evaluate any incoming HTTP request or L4 connection against the Routes you have configured and try to find a matching one.
+1. If multiple Routes match, the {{site.base_gateway}} router then orders all defined Routes by their priority and uses the highest priority matching Route to handle a request.
+1. If a given request matches the rules of a specific route, {{site.base_gateway}} will
+run any global, Route, or Service [plugins]() before it proxies the request. They are run in the following order: Route, Service <!--when does the global config get run?--> These configured plugins will run their `access` phase, which you can find more
+information about in the [Plugin development guide][plugin-development-guide].
```

Comment:
For reviewers: I don't think we added the autogenerated PDK yet. How do we want to handle links to autogenerated pages?